src/machine/behavior/is/mod.rs

```diff
@@ -23,14 +23,17 @@ pub mod system;
 
 use machine::state::State;
 
-pub fn get(_opcode: u8) -> Semantic {
-    unimplemented!();
-    // SEMANTICS[opcode]
+pub fn get(opcode: u8) -> Semantic {
+    SEMANTICS[opcode as usize]
+}
+
+pub fn instruction_name(opcode: u8) -> &'static str {
```

Comment:
It think, the function names `get` and `instruction_name` don't fit together.

Thoughts for a name schemata:
1. `get` maps the opcode to the semantic or functionality.
2. `instruction_name` maps the opcode to the symbolic name.

Maybe we can implement on `get()`-function that returns a tuple?!

What do the other think?